tests/functional.rs

```diff
@@ -1698,13 +1701,17 @@ async fn monero_syncer_broadcast_tx_test() {
 }
 
 async fn setup_monero() -> (monero_rpc::RegtestDaemonClient, monero_rpc::WalletClient) {
-    let dhost = env::var("MONERO_DAEMON_HOST").unwrap_or("localhost".into());
-    let daemon_client = monero_rpc::RpcClient::new(format!("http://{}:18081", dhost));
-    let daemon = daemon_client.daemon();
-    let regtest = daemon.regtest();
-    let whost = env::var("MONERO_WALLET_HOST_1").unwrap_or("localhost".into());
-    let wallet_client = monero_rpc::RpcClient::new(format!("http://{}:18083", whost));
-    let wallet = wallet_client.wallet();
+    let conf = config::TestConfig::parse();
+
+    let client = monero_rpc::RpcClient::new(format!("http://{}", conf.monero.daemon));
+    let regtest = client.daemon().regtest();
+
+    let client = monero_rpc::RpcClient::new(format!(
+        "http://{}",
```

Comment:
I think the url should be moved to the config as well. Might save a line here and there and to me it feels like it is part of the configuration.

Reply:
Done in b16d35c9d53330344a0a101502ef4bea0788a3e6